src/handbook/product/metrics.md

```diff
@@ -16,9 +16,21 @@ Aligned with our value of [Results](../company/values.md#📈-results), we are s
 #### Continuous learning and improvement
 Consistent with [Iterative Improvement](../company/values.md#🔁-iterative-improvement), we believe even the smallest changes can contribute to our growth. We're committed to consistently reassessing, learning, and evolving based on the insights our data provides.
 
-## Current Metrics
+## Leading Product Metric
 
-Each metric we track provides unique insights into different facets of our product. These metrics validate or disprove product hypotheses, guide product development, and assist in informed decision-making regarding feature improvements. The ultimate goal is always to create a product that fulfills the needs and preferences of our users. We focus on several key metrics that enable us to understand our product's use and the adoption of its various features. Here is a list of the current key metrics we track:
+The cornerstone of our department's evaluation is encapsulated in the Leading Product Metric, namely the Weighted Customer Index (WCI). This metric serves as our North Star, guiding strategic decisions and operational improvements. The WCI is calculated to reflect the value and distribution of our customer base across different product tiers.
+
+The formula for calculating the WCI is as follows:
+
+```Weighted Customer Index = Number of Starter Tier customers + (Number of Teams Tier customers * 4) + (Number of Enterprise Tier customers * 8)```
```

Comment:
```suggestion
```
Weighted Customer Index = Number of Starter Tier customers + (Number of Teams Tier customers * 4) + (Number of Enterprise Tier customers * 8)
```
```